src/sass/components/ATATFileUpload.scss

```diff
@@ -0,0 +1,32 @@
+.atat-file-upload,
+.atat-file-upload.v-input--is-focused.primary--text{
+    background-image: url("data:image/svg+xml,%3csvg width='100%25' height='100%25' xmlns='http://www.w3.org/2000/svg'%3e%3crect width='100%25'  height='100%25' fill='%23fbfbfb' stroke='%23544496' stroke-width='2' stroke-dasharray='10' stroke-dashoffset='2' stroke-linecap='square'/%3e%3c/svg%3e") !important;
```

Comment:
pls change `background-image` to 
```
url("data:image/svg+xml,%3csvg width='100%25' height='100%25' xmlns='http://www.w3.org/2000/svg'%3e%3crect width='100%25' height='100%25' fill='none' rx='8' ry='8' stroke='%2361686c' stroke-width='2' stroke-dasharray='8%2c 8' stroke-dashoffset='0' stroke-linecap='square'/%3e%3c/svg%3e") !important;
```

Reply:
- [ ] Done, except for the fill color.  Leaving that in b/c on the FIGMA is offwhite or #fbfbfb.
![image](https://user-images.githubusercontent.com/84856468/166853845-f5c794a8-f145-4518-a7e0-fef19ac33ad8.png)

Reply:
ah yes, I was focusing only on the border - forgot the background color